tests/test_qs.py

```diff
@@ -101,6 +102,37 @@ def test_prefetch_forward_relationship(self):
         with self.assertNumQueries(0):
             self.assertEqual(widgets[0].owner.name, "test owner")
 
+    def test_prefetch_forward_relationship_only_loads_pk_by_default(self):
+        Widget.objects.create(
+            name="test widget", owner=Owner.objects.create(name="test owner")
+        )
+
+        prepare = qs.prefetch_forward_relationship("owner", Owner.objects.all())
+
+        with CaptureQueriesContext(connection) as capture:
+            list(prepare(Widget.objects.all()))
+
+        self.assertEqual(len(capture.captured_queries), 2)
+
+        self.assertEqual(
+            capture.captured_queries[0]["sql"],
+            "SELECT "
+            '"tests_widget"."id", '
+            '"tests_widget"."owner_id" '
+            "FROM "
+            '"tests_widget"',
```

Comment:
nit: Can't you use single quote for every chunk of this long string? flipping back and forth each line is pretty terrible

Reply:
Sadly not, this is Black doing the formatting. It might be that there's a nicer way to handle these multiline SQL strings..